lib/widgets/message_list.dart

```diff
@@ -915,10 +913,11 @@ class MessageWithPossibleSender extends StatelessWidget {
                   const SizedBox(width: 8),
                   Flexible(
                     child: Text(message.senderFullName, // TODO get from user data
-                      style: const TextStyle(
+                      style: TextStyle(
                         fontFamily: 'Source Sans 3',
                         fontSize: 18,
                         height: (22 / 18),
+                        color: const HSLColor.fromAHSL(1, 0, 0, 0.2).toColor(),
```

Comment:
msglist: Set message-sender-name color to match web

    …
    That color:
      const HSLColor.fromAHSL(1, 0, 0, 0.15).toColor()
    doesn't match web, it turns out. (Or the old Figma, which uses
    true black.)

It'd be good to have this change in screenshots, too.